src/components/FilterPane/FilterPane.jsx

```diff
@@ -28,38 +28,20 @@ import {
   TieredStyledClickableArea,
 } from './FilterPane.styles'
 import { filterPane } from '../../constants/language'
-import { URL_PARAMS, COLLECTION_METHODS } from '../../constants/constants'
-import { useEffect, useState, useCallback, useContext } from 'react'
-import { useLocation, useNavigate } from 'react-router-dom'
+import { FilterProjectsContext } from '../../context/FilterProjectsContext'
 import { IconClose, IconPlus } from '../../assets/icons'
 import { IconUserCircle, IconMinus } from '../../assets/dashboardOnlyIcons'
-import { FilterProjectsContext } from '../../context/FilterProjectsContext'
+import { MermaidMenuItem, MermaidSelect } from '../generic/MermaidSelect'
+import { URL_PARAMS, COLLECTION_METHODS } from '../../constants/constants'
 import { useAuth0 } from '@auth0/auth0-react'
+import { useEffect, useState, useCallback, useContext } from 'react'
+import { useLocation, useNavigate } from 'react-router-dom'
 
 const deleteIconSize = {
   height: '15px',
   width: '15px',
 }
 
```

Comment:
Changes in this file are just me making a reusable component for the select input